app/javascript/miq-redux/redux-typings.ts

```diff
@@ -40,5 +40,5 @@ export type ReduxStore = Store<AppState>;
  */
 export interface ReduxApi {
   store: ReduxStore;
-  addReducer(appReducer: AppReducer): Unsubscribe;
```

Comment:
^ unnecessary import on line 1

Reply:
But actually.. this whole file is now dead since there is no typescript code touching reducers.. Not sure what the plans are though..

Reply:
Yeah i wanted to delete it too, but as you mentioned there is still that file that uses it (it just provides dev hints in text editors). I would like to get rid of the typescript in near future (probably after this PR) and use just ES6. So maybe leave it for some future PR?